### PR TITLE
chore: add Tailwind

### PR DESCRIPTION
**Pull request by astone123**
This PR adds [Tailwind CSS](https://tailwindcss.com/) so that we can use the utility classes that it provides. It also refactors the use of all `style` props to use Tailwind classes instead.

### CODE REVIEW

**Comment by wweitzel** [comment]
I don't think I want to add another css lib since we use bootstrap currently and it has utility classes just like this.

File: src/components/Header.tsx [hunk 1]
```diff
@@ -29,7 +24,7 @@ export function Header({selectedTheme, onClick}: HeaderProps) {
 
   return (
     <div className="d-flex justify-content-center">
-      <img style={logoStyle} src={logo} onClick={onClick} alt="logo" />
+      <img className="cursor-pointer h-[250px]" src={logo} onClick={onClick} alt="logo" />
```
Comment (wweitzel) [inline src/components/Header.tsx:27]: Could do the cursor with existing bootsrap. https://getbootstrap.com/docs/4.5/content/reboot/#pointers-on-buttons

Not sure about pixel height though

File: src/components/Input.tsx [hunk 2]
```diff
@@ -7,12 +7,11 @@ interface InputProps {
 
 function Input({label, placeholder, value, onInput}: InputProps) {
   return (
-    <div style={{width: '100%'}}>
+    <div className="w-[100%]">
```
Comment (wweitzel) [inline src/components/Input.tsx:10]: bootstrap way https://getbootstrap.com/docs/5.0/utilities/sizing/#relative-to-the-parent

File: src/components/Input.tsx [hunk 3]
```diff
@@ -7,12 +7,11 @@ interface InputProps {
...
       <label className="form-label">{label}</label>
-      <div style={{borderRadius: '20px'}} className="shadow-sm">
+      <div className="shadow-sm rounded-full">
```
Comment (wweitzel) [inline src/components/Input.tsx:12]: https://getbootstrap.com/docs/5.0/utilities/borders/#border-radius

File: src/components/Header.tsx (hunk 1 above)
Comment (astone123) [reply src/components/Header.tsx:27]: As far as I can tell you can only do percentages with bootstrap. Can't specify pixels

Comment (wweitzel) [reply src/components/Header.tsx:27]: yes, looks like it

File: src/pages/Goals.tsx [hunk 4]
```diff
@@ -204,7 +195,7 @@ function Goals() {
           <br></br>
 
           <div className="d-flex">
-            <div className="flex-grow-1" style={{flexBasis: '0'}}>
+            <div className="flex-grow-1 basis-0">
```
Comment (wweitzel) [inline src/pages/Goals.tsx:198]: Does not look like bootstrap has a thing for this

**Comment by wweitzel** [comment]
Yeah I don't know @astone123. It seems there are a few things that tailwind can do that bootstrap cannot. Still not sure if we want to mix these together.

**Comment by astone123** [comment]
@wweitzel okay I guess we should stick to bootstrap

**Comment by wweitzel** [comment]
yeah i guess. im not married to bootstrap at all. just don't want two css libs mixing in a tiny project like this

**Pull request closed without merging**